React/CoreModules/RCTDevSettings.h

```diff
@@ -28,6 +28,11 @@
  */
 - (id)settingForKey:(NSString *)key;
 
+/**
```

Comment:
any changes to RN code should be tagged so we know it's an intentional diff from upstream.

```
// [TODO(macOS ISS#2323203)
some code here
// TODO(macOS ISS#2323203)]
```

Reply:
This applies to the other changes too